### PR TITLE
Updates for Dash 2.0

### PR DESCRIPTION
**Pull request by AnnMarieW**
**This PR resets dash-labs for new projects based on dash>= v2.0.0 and dash-bootstrap-components >= v1.0.0**

**Tasks completed:**
- [x] Removed all code and tests based on dash < V2.0.0
- [x] All documentation from past projects updated with project status.
- [x]  Update READM.md
- [x] Update CHANGELOG
- [x] Update requirements.txt
- [x] Bumped version to 1.0
- [x] Added the Multi-Page Dash App Plug-in
- [x] Fixed bugs in `plugin/pages.py`  that caused errors if `/assets` or  images in the `/assets` files did not exist.
- [x] Added 2 multi-page app examples
- [x] Added new chapter to documentation for multi-page apps
- [x] Added support for nested folders

**To Do:**
- [ ] Make sure all links work after merge
- [ ] Deploy
- [ ] Announcement on the forum

**Phase II**
- [ ] open issues for improvements to the docs
- [ ] open issue for adding tests
- [ ] open issues for feature requests from community forum discussion
I have the following project  ready to add when/if this is PR is merged.
- [ ] Dashdown - markdown that runs Dash code snippets
- [ ] Code snippet app  - Like [this one](https://dash.gallery/python-docs-dash-snippets/) but updated for Dash 2.0
- [ ] Two AOI components for switching Bootstrap themes in an app.
- [ ] A stylesheet that styles dash-core-components and the DataTable with Bootstrap theme colors.
- [ ] New low code layout templates

**Questions for Reviewers:**
- I changed the name `pages_plugin.py` to just `pages.py` so it could be added like this:
  `plugins=[dl.plugins.pages]`  
Is that acceptable?

- When this is added to Dash, will it be possible to add the container like this?:
  `dash.page_container`
currently, it needs to be imported from the plugin like this:
`dl.plugins.page_container`

### CODE REVIEW

File: dash_labs/version.py [hunk 1]
```diff
@@ -1 +1 @@
-__version__ = "0.4.0"
+__version__ = "1.0.0"
```
Comment (chriddyp) [inline dash_labs/version.py:1]:
```suggestion
__version__ = "1.0.0rc1"
```

That way you can test a pypi publish first :)

File: docs/02-CallbackEnhancements.md [hunk 2]
```diff
@@ -1,3 +1,18 @@
...
+> ## Status: Callback Enhancements
+> #### These callback enhancements were added to Dash 2.0 🎉
+> #### See the [Flexible Callback Signature section](https://dash.plotly.com/flexible-callback-signatures) in the Dash documentation.
+
+
+```diff
+- ----------------------------------------------------------------------------------
+-  This documentation describes code in a previous version of dash-labs (v0.4.0) 
+-  and is included here for legacy purposes only.
+- ----------------------------------------------------------------------------------
```
Comment (chriddyp) [inline docs/02-CallbackEnhancements.md:11]: Should we include something like:
```
You can install v0.4.0 with:
pip install dash-labs==0.4.0
```

?

File: dash_labs/version.py (hunk 1 above)
Comment (AnnMarieW) [reply dash_labs/version.py]: good idea. done.

**Review by chriddyp — APPROVED** [review]
💃🏼

**Pull request merged**